src/Extensions/DataListExtension.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace SilverStripe\Versioned\Extensions;
+
+use RuntimeException;
+use SilverStripe\Core\Extension;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Versioned\Versioned;
+
+class DataListExtension extends Extension
+{
+    public function updatePrepopulateCaches(string &$dataClass, array &$ids): void
```

Comment:
```suggestion
    public function updatePrepopulateCaches(string $dataClass, array $ids): void
```
We don't actually want to change the upstream values of these, so we shouldn't pass by reference.

Reply:
Updated